resdk/resolwe.py

```diff
@@ -267,7 +267,7 @@ def run(self, slug=None, input={}, descriptor=None,  # pylint: disable=redefined
         :param dict input: Input values
         :param dict descriptor: Descriptor values
         :param str descriptor_schema: A valid descriptor schema slug
-        :param list collection: Collection into which data object should be included
+        :param int collection: Collection (id) into which data object should be included
```

Comment:
It can be both: `Collection` resource or colection id. This might be longer but also more explicit: "Collection resource or it's id into which data object should be included"

Reply:
Good suggestion. I edited to include the resource as well. Not sure if there's a better way of listing several possible values.